src/Okta.Idx.Sdk/IdxClient.cs

```diff
@@ -237,6 +240,78 @@ internal async Task<IIdxResponse> IntrospectAsync(IIdxContext idxContext, Cancel
                 request, cancellationToken).ConfigureAwait(false);
         }
 
+        /// <inheritdoc/>
+        public async Task<WidgetSignInResponse> StartWidgetSignInAsync(CancellationToken cancellationToken = default)
+        {
+            var idxContext = await this.InteractAsync();
+            return new WidgetSignInResponse
+            {
+                IdxContext = idxContext,
+                SignInWidgetConfiguration = new SignInWidgetConfiguration(this.Configuration, idxContext),
+            };
+        }
+
+        /// <inheritdoc/>
+        public async Task<TokenResponse> RedeemInteractionCodeAsync(IIdxContext idxContext, string interactionCode, CancellationToken cancellationToken = default)
+        {
+            try
+            {
+                Uri issuerUri = new Uri(Configuration.Issuer);
+                Uri tokenUri = new Uri(IdxUrlHelper.GetNormalizedUriString(issuerUri.ToString(), "v1/token"));
```

Comment:
What happens here if the developer passes an issuerUri without the trailing /. Can we add a check in here to make sure it includes the / or are we ok with failing due to malformed URL?

Reply:
The default behavior of Uri.ToString() adds the trailing slash.